source/modulo_controllers/controller_descriptions/modulo_controllers_controller_interface.json

```
@@ -2,9 +2,10 @@
   "$schema": "https://docs.aica.tech/schemas/1-3-0/controller.schema.json",
   "name": "Controller Interface",
   "description": {
-    "brief": "Base controller class to combine ros2_control, control libraries and modulo"
+    "brief": "Base controller class to combine ros2_control, control libraries and modulo" // todo: to be updated
```

I would propose something like:

```suggestion
    "brief": "Opinionated controller interface class that includes custom parameters for derived controllers"
```